<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowShowRobot</class>
 <widget class="QMainWindow" name="MainWindowShowRobot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>1046</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simox - VirtualRobot - Show Robot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frameViewer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>841</width>
          <height>1092</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButtonReset">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonLoad">
           <property name="text">
            <string>Select Robot File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonReLoad">
           <property name="text">
            <string>Reload Robot File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxJoints">
           <property name="title">
            <string>Joints</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0" colspan="2">
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="5" column="0" colspan="2">
                <widget class="QSlider" name="horizontalSliderPos">
                 <property name="maximum">
                  <number>1000</number>
                 </property>
                 <property name="pageStep">
                  <number>50</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Joint Value</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="labelMinPos">
                 <property name="text">
                  <string>-1.57</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QLabel" name="labelMaxPos">
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="text">
                  <string>1.57</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QComboBox" name="comboBoxRobotNodeSet"/>
               </item>
               <item row="2" column="0" colspan="2">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Select RobotNode</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLCDNumber" name="lcdNumberJointValue">
                 <property name="smallDecimalPoint">
                  <bool>true</bool>
                 </property>
                 <property name="digitCount">
                  <number>5</number>
                 </property>
                 <property name="value" stdset="0">
                  <double>0.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" colspan="2">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Select RobotNodeSet</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="0" colspan="2">
                <widget class="QCheckBox" name="checkBoxShowCoordSystem">
                 <property name="text">
                  <string>RobotNode's Coord System</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0" colspan="2">
                <widget class="QComboBox" name="comboBoxJoint"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxEEF">
           <property name="title">
            <string>End Effector</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="horizontalSpacing">
             <number>0</number>
            </property>
            <property name="verticalSpacing">
             <number>3</number>
            </property>
            <item row="0" column="0" colspan="3">
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="0" colspan="2">
                <widget class="QComboBox" name="comboBoxEndEffector"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Shape</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="comboBoxEndEffectorPS"/>
               </item>
               <item row="2" column="0">
                <widget class="QPushButton" name="pushButtonClose">
                 <property name="text">
                  <string>Close</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="pushButtonOpen">
                 <property name="text">
                  <string>Open</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxDisplayOpt">
           <property name="title">
            <string>Display options</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0" colspan="2">
             <widget class="QWidget" name="widget" native="true">
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="6" column="0">
                <widget class="QCheckBox" name="checkBoxPhysicsCoM">
                 <property name="text">
                  <string>CoM</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QCheckBox" name="checkBoxPhysicsInertia">
                 <property name="text">
                  <string>Inertia</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0" colspan="2">
                <widget class="QCheckBox" name="checkBoxRobotCoordSystems">
                 <property name="text">
                  <string>Coordinate systems</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0" colspan="2">
                <widget class="QCheckBox" name="checkBoxRobotSensors">
                 <property name="text">
                  <string>Sensors</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0" colspan="2">
                <widget class="QCheckBox" name="checkBoxStructure">
                 <property name="text">
                  <string>Robot structure</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" colspan="2">
                <widget class="QCheckBox" name="checkBoxFullModel">
                 <property name="text">
                  <string>IV Model</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Full model options</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" colspan="2">
                <widget class="QCheckBox" name="checkBoxColModel">
                 <property name="text">
                  <string>Collision model</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ExportVRML20">
           <property name="toolTip">
            <string>Export to single VRML file</string>
           </property>
           <property name="text">
            <string>Export VRML</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ExportXML">
           <property name="toolTip">
            <string>Export to simox XML format</string>
           </property>
           <property name="text">
            <string>Export XML</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxTriangles">
           <property name="title">
            <string>Triangles</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <layout class="QGridLayout" name="gridLayout_8">
               <item row="1" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Total</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QLabel" name="labelTriColRNS">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="labelTriVisuRNS">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="labelTriVisuTotal">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QLabel" name="labelTriColJoint">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Joint</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="labelTriColTotal">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLabel" name="labelTriVisuJoint">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>RobotNodeSet</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>Visu</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>Col</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="openDiffIKButton">
           <property name="text">
            <string>Open Diff IK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxDistance">
           <property name="title">
            <string>Distance</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="1" column="0">
             <widget class="QWidget" name="widget_5" native="true">
              <layout class="QGridLayout" name="gridLayout_10">
               <item row="2" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Distance</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QDoubleSpinBox" name="doubleSpinBoxDistancePtZ">
                 <property name="minimum">
                  <double>-2000.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>2000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" colspan="4">
                <widget class="QCheckBox" name="checkBoxDistToPtEnabled">
                 <property name="text">
                  <string>Show Point Distance</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBoxDistancePtX">
                 <property name="minimum">
                  <double>-2000.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>2000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>xyz</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1" colspan="3">
                <widget class="QLabel" name="labelDistancePtDist">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QDoubleSpinBox" name="doubleSpinBoxDistancePtY">
                 <property name="minimum">
                  <double>-2000.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>2000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>groupBoxDisplayOpt</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>658</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>984</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBoxJoints</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>668</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBoxEEF</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_3</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>697</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBoxTriangles</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_4</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>661</x>
     <y>817</y>
    </hint>
    <hint type="destinationlabel">
     <x>671</x>
     <y>837</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBoxDistance</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_5</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>612</x>
     <y>929</y>
    </hint>
    <hint type="destinationlabel">
     <x>651</x>
     <y>960</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
